test/remote/too_many_bad_logins.js

```diff
@@ -14,7 +14,8 @@ var config = {
     port: 7000
   },
   limits: {
-    rateLimitIntervalSeconds: 1
+    rateLimitIntervalSeconds: 1,
+    badLoginLockoutIntervalSeconds: 1
```

Comment:
Strictly speaking, this line isn't used for anything as far as I can tell. I added it for consistency.